modules/core/shared/src/main/scala/freestyle/free/internal/ScalametaUtil.scala

```diff
@@ -65,11 +65,6 @@ object ScalametaUtil {
       case _           => true
     }
 
-    def isStackSafe: Boolean = mods.exists {
-      case mod"@stacksafe" => true
```

Comment:
Thanks; we still have a reference to `@stacksafe` a few lines above in `filtered()` to remove.